packages/cfpb-forms/src/organisms/Multiselect.js

```diff
@@ -0,0 +1,500 @@
+// Required modules.
+//import { checkDom, setInitFlag } from '../modules/util/atomic-helpers';
```

Comment:
Should these functions be deleted or are they intentionally left in this file as a reminder for a future migration?

Reply:
I pushed up https://github.com/cfpb/design-system/pull/1096/commits/3f40ff16e9b0f8c6e18e63187120de7b73557de3 to remove these, though we should work out what we want to do here. In components like this that use the `atomic-helpers` utility in cf.gov, the element gets `data-js-hook="state_atomic_init"` set on the dom so that the component isn't initialized twice. Here in the design system, components that inherit from the `AtomicComponent` class set `data-bound="true"` on the dom. Which do we want to use? Well, I dunno. `AtomicComponent` has a [totally different structure](https://github.com/cfpb/design-system/blob/main/packages/cfpb-expandables/src/Expandable.js#L12), so would require a re-write of the multiselect. Bringing in `atomic-helpers` to the DS means we have two atomic utilities in the DS, which doesn't make sense.